tests/projects/test_schema.py

```diff
@@ -19,28 +14,5 @@ def test_schema(snapshot: Snapshot):
 
 
 def test_schema_with_inheritance(snapshot: Snapshot):
-    @strawberry_django.type(Project)
-    class ProjectTypeSubclass(ProjectType): ...
-
-    @strawberry_django.type(Milestone)
-    class MilestoneTypeSubclass(MilestoneType): ...
-
-    @strawberry_django.input(Issue)
-    class IssueInputSubclass(IssueInput): ...
-
-    @strawberry.type
-    class Query:
-        project: Optional[ProjectTypeSubclass] = strawberry_django.node()
-        milestone: Optional[MilestoneTypeSubclass] = strawberry_django.node()
-
-    @strawberry.type
-    class Mutation:
-        create_issue: IssueType = mutations.create(
-            IssueInputSubclass,
-            handle_django_errors=True,
-            argument_name="input",
-        )
-
-    schema = strawberry.Schema(query=Query, mutation=Mutation)
     snapshot.snapshot_dir = SNAPSHOTS_DIR
-    snapshot.assert_match(str(schema), "schema_with_inheritance.gql")
```

Comment:
Why did you have to change this?

Reply:
Oh, that `schema_inherited.py` is not actually used anywhere (maybe it is a left over from some old attempt)

I think the ideal here is to remove that file and have the test "as is", as it is usually better to have everything defined inside the test itself.

I'll send a commit to main removing that file